### ISSUE TITLE
Dokka 0.9.13 crashes JVM on build.

### ISSUE THREAD

**Issue by ajalt** [Kotlin/dokka#135]
I recently upgraded the Dokka version on an android library that I publish through [jitpack](jitpack.io).

It appears that Dokka is crashing the JVM when run on the Jitpack servers. I can not reproduce the crash on my local machine, but it happens on every build on jitpack.

Here's a log with a stacktrace and a thread dump that contains the error: 
https://jitpack.io/com/github/jitpack-io/flexadapter/e9d8686/build.log

And here's another log with just a stacktrace, but that uses the gradle daemon:
https://jitpack.io/com/github/ajalt/flexadapter/140483df580a998b301d32ac3a37a67b793967a0/build.log

Here are my gradle files: 
https://github.com/ajalt/flexadapter/blob/master/build.gradle
https://github.com/ajalt/flexadapter/blob/master/flexadapter/build.gradle

Let me know if there's any more information I can provide.

**Comment by semoro** [comment]
I noticed such crashes while development of new Dokka build configuration, I will check for workaround and answer soon.

**Comment by semoro** [comment]
This happens when you loading plugin when working directory != directory with gradlew

**Comment by gurunars** [comment]
@ajalt sorry, which JDK do you happen to have locally? I, unfortunately, fully reproduce the issue. Is it a Sun one? Or OpenJDK?

**Comment by ajalt** [comment]
From the log that I posted:
```
# JRE version: Java(TM) SE Runtime Environment (8.0_101-b13) (build 1.8.0_101-b13)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.101-b13 mixed mode linux-amd64 compressed oops)
```

Is that what you were looking for?

**Comment by gurunars** [comment]
@ajalt 

    openjdk version "1.8.0_76-release"
    OpenJDK Runtime Environment (build 1.8.0_76-release-b03)
    OpenJDK 64-Bit Server VM (build 25.76-b03, mixed mode)

Not the same, but close. The issue is reproducible locally.

**Comment by gurunars** [comment]
Tested on the latest Oracle JDK. Fails as well. Sdk version:

    java version "1.8.0_121"
    Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
    Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)

**Comment by semoro** [comment]
Fixed in [0.9.14](https://github.com/Kotlin/dokka/releases/tag/0.9.14)
